### PR TITLE
CSS and image fixes

### PR DESCRIPTION
**Pull request by FaisalAbid**
Fixes for

> Section headlines: 
> Could we please reduce to line-height:1.2; for the section headlines
> Can we please update the last headline to title case like the others: ‘Fast on All Platforms’ (at the intro up top as well)
> The list of proof points below the headlines are offset too far to the right, can we please make sure the selection bullet point visually left aligns with the left edge of the section title?
> 
> Images
> We had an edit to one of the illustrations. I also noticed an issue on my end where some of the images had a white line at the bottom. I’ve updated all the images in the ‘assets’ link on drive. Could please replace these with the new set? I have not changed the naming at all, so hopefully these are easy to switch out.
> 
> Mobile
> For the section headlines and proof points, could we increase the right and left margin size slightly (~5px /side)?
> Can you please check that the circle icons above the section headlines are sized at 68x68px. These look a bit small on mobile
> Can you please check the size of the text under ‘Made by Google’ is 18px. This looks a bit large on mobile. 
> Can you please make sure we include a right and left margin for the ‘Made by Google’ section. The text runs to the edges on my browser.
> Hero: please see the mobile design comp on Gallery for the hero image. We should add some space (color: #13202F) above and below the video to give it more presence at the top of the page
>

### CODE REVIEW

**Comment by FaisalAbid** [comment]
Staged ->  https://kw-staging-dartlang-2.firebaseapp.com

**Comment by kwalrath** [comment]
I'm not sure who gave the feedback on the capitalization, but I disagree on using title case. (Shocking, I know. :) If we really do want to do that, please discuss it with me and I'll make it happen correctly.

In general, HTML/CSS implementation work should go to Faisal, and word stuff should go to me.

**Comment by FaisalAbid** [comment]
I saw that you had changed it! I didn't know if it was before or after the change was requested to me

**Comment by kwalrath** [comment]
> I saw that you had changed it! I didn't know if it was before or after the change was requested to me

Yeah, I have no idea. But whatever happens, you shouldn't need to deal with capitalization and punctuation and the like. That's my job! (Cue [Galaxy Quest quote](https://www.youtube.com/watch?v=S4EyF2H_Tjg).)

**Comment by FaisalAbid** [comment]
@kwalrath Sounds good! I reverted it back to what you had

File: src/_assets/css/_dash.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@ $dash-dark-black: #4a4a4a;
 $dash-highlight: #1967d2;
 $dash-off-black: #12202f;
 $dark-darker-black: #0d1520;
-$dash-header-bg: #17212f;
+$dash-header-bg: #12202f;
```
Comment (johnpryan) [inline src/_assets/css/_dash.scss]: On macOS the background color of the image seems to be #17212f. This change causes a slight issue with the background:

Screen Shot 2019-05-06 at 10 51 09 AM

**Comment by messi01** [comment]
LGTM

**Comment by messi01** [comment]
LGTM

**Comment by kevmoo** [comment]
Replaced by https://github.com/dart-lang/site-www/pull/1573

File: src/_assets/css/_dash.scss (hunk 1 above)
Comment (kwalrath) [reply src/_assets/css/_dash.scss]: @johnpryan is this still an issue?

Comment (johnpryan) [reply src/_assets/css/_dash.scss]: yeah, I can make a PR to master

Comment (johnpryan) [reply src/_assets/css/_dash.scss]: PR here: https://github.com/dart-lang/site-www/pull/1581

**Pull request closed without merging**